### ISSUE TITLE
Method not found: 'Void MailKit.MailTransport.Send(MimeKit.MimeMessage, System.Threading.CancellationToken, MailKit.ITransferProgress)'.

### ISSUE THREAD

**Issue by neozhu** [lukencode/FluentEmail#326]
services.AddFluentEmail(mailSettings.From)
                .AddRazorRenderer()
  // this configuration is working
        .AddSmtpSender(new System.Net.Mail.SmtpClient()
        {
            Host = mailSettings.Host,
            Port = mailSettings.Port,
            EnableSsl = mailSettings.UseSsl,
            Credentials = new System.Net.NetworkCredential(mailSettings.UserName, mailSettings.Password)
        });
       // this configuration with Mailkit not working.
        //.AddMailKitSender(new FluentEmail.MailKitSmtp.SmtpClientOptions()
        //{
        //    RequiresAuthentication = !string.IsNullOrEmpty(mailSettings.UserName),
        //    UsePickupDirectory = false,
        //    Password = mailSettings.Password,
        //    Port = mailSettings.Port,
        //    Server = mailSettings.Host,
        //    User = mailSettings.UserName,
        //    UseSsl = mailSettings.UseSsl,
        //});

Method not found: 'Void MailKit.MailTransport.Send(MimeKit.MimeMessage, System.Threading.CancellationToken, MailKit.ITransferProgress)'.
why?

**Comment by ghost** [comment]
Yes. I have exact same issue.  I'm very surprised that the author @lukencode has not responded at all since April. I guess it's time not to use this nugget packages

**Comment by tigrin** [comment]
For me, this issue happens when I have [MailKit ](https://github.com/jstedfast/MailKit) version 3.0.0 or greater installed. 

It looks like MailKit 3.0.0 changed the signature to return a string instead of a void. [Link to commit](https://github.com/jstedfast/MailKit/commit/e32bb65b91e701246b999dbb33b760f0fb84d633)

**Comment by AdisonCavani** [comment]
This is a duplicate of #296
You can close the issue and move this conversation to issue above.
